tests/e2e/snackbar.wdio-spec.js

```diff
@@ -31,18 +31,20 @@ describe('Snackbar', () => {
   it('should display the snackbar with the syncing messages and then hide it', async () => {
     await commonPage.openHamburgerMenu();
     await (await commonPage.syncButton()).click();
-    await (await commonPage.activeSnackbar()).waitForDisplayed();
```

Comment:
I think this line was the culprit... from the [WDIO docs](https://webdriver.io/docs/api/element/waitForDisplayed):

> As opposed to other element commands WebdriverIO will not wait for the element to exist to execute this command.

What this meant was if this line was executed before the `active` class was applied to the snackbar then the selector failed to find the element and it was not retried.

The change I've made is relying on there only ever being one snackbar element, so if you don't check for the `active` class it's sure to find the element. Then await the element being displayed and the message being updated.